doc/changelog/1.21.0-changelog.rst

```diff
@@ -564,7 +564,7 @@ A total of 581 pull requests were merged for this release.
 * `#18670 <https://github.com/numpy/numpy/pull/18670>`__: BUG: Fix small issues found with pytest-leaks
 * `#18676 <https://github.com/numpy/numpy/pull/18676>`__: MAINT: Implement new style promotion for `np.result_type`, etc.
 * `#18679 <https://github.com/numpy/numpy/pull/18679>`__: BUG: Changed METH_VARARGS to METH_NOARGS
-* `#18680 <https://github.com/numpy/numpy/pull/18680>`__: Docs: simd-optimizations.rst: fix typo (basline ~> baseline)
+* `#18680 <https://github.com/numpy/numpy/pull/18680>`__: Docs: simd-optimizations.rst: fix typo (baseline ~> baseline)
```

Comment:
This is also not desirable, the PR title was clearly describing a typo fix, with the before/after representations

Reply:
fixed. Sorry